### ISSUE TITLE
UIViewController.navigationItem.leftBarButtonItem.title and image

### ISSUE THREAD

**Issue by comm1x** [schibsted/layout#67]
After adding property `navigationItem.leftBarButtonItem.title` to UIViewController, view stops initializing. Method `loadLayout()` not crashes, but do nothing. All outlets is nil. After removal this line all is ok.

Version of Layout is 0.5.4

```
<UINavigationController outlet="nav">
    <UIViewController
        title="Title"
        backgroundColor="white"
        navigationItem.leftBarButtonItem.title="List">
```
------------------------------------------
Also I want also to suggest force throw exceptions, if something went wrong. After upgrading from 0.4.20 to 0.5.4, my project start fails on run, without any error description in logs. It turned out that I have 2 errors: use `UILabel.color` instead of `UILabel.textColor` and this issue with `navigationItem`. So, let's explicit throw or log error.

**Comment by nicklockwood** [comment]
Hey @comm1x, thanks for reporting this. There're a couple of bugs here, one with setting the `leftBarButtonItem.title` and another with the error not being reported. I've fixed both of these for the next release.

As for forcing errors to be thrown, the issue in this case was that the error was being silently swallowed, so forcing it to be thrown wouldn't help in this case.

For future reference though, if you ever have a situation where errors aren't being reported, try enabling break-on-swift-errors in Xcode, as that will often catch the error even if it doesn't get logged.

**Comment by nicklockwood** [comment]
Fixed in 0.5.6

**Comment by comm1x** [comment]
Hi Nick, I have 2 news for you. :) `navigationItem.leftBarButtonItem.title` successfully fixed 0.5.6, but I just noticed that `navigationItem.leftBarButtonItem.image` broken in 0.5.6  :) I think it was broken before, but I saw it only now.

After setting `image` property, I see red screen about `private or read-only in UINavigationController`.

![Screenshot](https://gyazo.com/b90c2837cdee164c0160dc5d4d0ebed6.png)

**Comment by nicklockwood** [comment]
D'oh! OK, I'll push another patch tomorrow.

**Comment by nicklockwood** [comment]
@comm1x if you don't mind using a pre-release version until the next update, the fix is here: https://github.com/schibsted/layout/tree/develop

**Comment by nicklockwood** [comment]
Fixed in 0.5.7
